README.md

```diff
@@ -1,2 +1,14 @@
 # fpm-registry
+
 Centralized registry of fpm packages
+
+## How to submit a new package
+
+Please submit a PR against this repository, add the new package into the file
+[registry.toml](./registry.toml). Explicitly list each version (or the latest
+git commit), using a syntax like:
```

Comment:
Do you mean you can specify a specific commit? Or do you mean just the default behavior of getting the master branch?

If the latter, we should rather say something like:

"Explicitly list each version using the `tag` keyword, otherwise the master branch will be used."

Reply:
Yes, the latter. Fixed in 9eb7ec1b23b2e96e80a12747d9a62d9fadd3179c.